src/main/java/salesforce/ui/pages/product/ProductLightningSkin.java

```diff
@@ -72,32 +53,34 @@ public boolean isActive() {
      * @return the description
      */
     public String getDescription() {
-        return getWebElementAction().getTextOnWebElement(descriptionTextArea);
+            return getWebElementAction().getTextOnWebElement(getDriver().findElement(descriptionTextArea));
```

Comment:
I think there is more space in the return

Reply:
i'll delete it